content/gherkin.md

```diff
@@ -72,19 +73,19 @@ Feature: Guess the word
   Example: Maker starts a game
 ```
 
-The name and the description have no special meaning to Cucumber. Their purpose is to provide
+The name and the Optional description have no special meaning to Cucumber. Their purpose is to provide
```

Comment:
lower case "Optional" -> "optional" (although it's implied by the rest of the sentence, both name and description are optional

Reply:
don't think you need optional here, prefer the original

Reply:
Same :)